pandas/core/groupby/groupby.py

```diff
@@ -1751,7 +1749,7 @@ def nth(self, n: Union[int, List[int]], dropna: Optional[str] = None) -> DataFra
             raise ValueError(
                 "For a DataFrame groupby, dropna must be "
                 "either None, 'any' or 'all', "
-                "(was passed {dropna}).".format(dropna=dropna)
+                f"(was passed {dropna})."
```

Comment:
@datapythonista do we have a preferred usage for cases where this line needs the "f" and the preceeding does not?  I could imagine we might want to put the "f" on all of them anyway

Reply:
I think this is correct - should just keep it on lines where needed

Reply:
Agree with @WillAyd, my preference is to just have the `f` were it's needed. Even in strings splitted in different lines.

Reply:
As described in [pep-498-concatenating-strings](https://www.python.org/dev/peps/pep-0498/#concatenating-strings), regular strings are concatenated at compile time, and f-strings are concatenated at run time.

I think what we should not concatenate regular strings with f-strings because this could lead to unexpected results.